File: JavaScript/JS Core/02.JS Advanced/19.Modules/16-Modules/04-people-and-posts/person.js
class Person {
    constructor(name, age) {
        this.name = name;
        this.age = age;
    }

    addToSelector(selector) {
        let personHTML =
            $('<div>')
                .addClass('person ' + this.name)
                .append($('<p>')
                    .addClass('name')
                    .text(this.name))
                .append($('<p>')
                    .addClass('age')
                    .text(this.age))
                .append($('<div>')
                    .addClass('posts ' + this.name));
        $(selector).append(personHTML);
    }
}

module.exports = Person;

File: Java/Java Fundamentals/02.Java OOP Basics - Jun 2018/05.02.Exercise Interfaces and Abstraction/src/p05BordeControl/Person.java
package p05BordeControl;


public abstract class Person implements Identificatable {
    private String name;
    private int age;
    private String id;


    Person(String name, int age, String id) {
        this.name = name;
        this.age = age;
        this.id = id;
    }

    Person(String name, String id) {
        this.name = name;
        this.id = id;
    }

    @Override
    public String getId() {
        return this.id;
    }
}
